spotify_connect_scrobbler/credentials.py

```diff
@@ -1,22 +1,19 @@
-from bson.objectid import ObjectId
-
-import json
-import os
-import pymongo
-
-
 class LastfmCredentials:
     """LastFM API credentials.
 
     Args:
         session_key (str): Session key returned by the authentication endpoint.
     """
 
-    def __init__(self, session_key):
+    def __init__(self, api_key, api_secret, session_key):
+        self.api_key = api_key
+        self.api_secret = api_secret
         self.session_key = session_key
 
     def todict(self):
-        return {'session_key': self.session_key}
+        return {'api_key': self.api_key,
+                'api_secret': self.api_secret,
```

Comment:
I totally agree. When I was coding this I was thinking it was user based, but is of course "our" service keys. Will fix this.